README.md

```diff
@@ -205,6 +205,7 @@ Found [Clone] 7 duplicated lines with 10 tokens:
     * `--skip-commit-uuid-validation` [default: false] - Force using a commit UUID even if it doesn't belong to the current git branch.
     * `--skip-uncommitted-files-check` [default: false] - Skip check for uncommitted files in the analysis directory
     * `--upload` [default: false] - Request to push results to Codacy
+    * `--upload-batch-size` [default: 50000] - Batch size for upload of results
```

Comment:
A small note: maybe the default should be smaller since 50k might break the max payload size.